src/functions/updateTechRecord.ts

```diff
@@ -10,12 +10,17 @@ export function updateTechRecordStatus(event: SQSEvent) {
 
     event.Records.forEach((record: SQSRecord) => {
         const test = JSON.parse(record.body);
-        const promise = UpdateTechRecordService.updateStatusByVin(test.vin,
+        const promiseUpdateStatus = UpdateTechRecordService.updateStatusByVin(test.vin,
             test.testStatus,
             test.testTypes.testResult,
             test.testTypes.testTypeId,
             test.newStatus);
-        promisesArray.push(promise);
+        if (test.euVehicleCategory) {
```

Comment:
A submitted test-result will always have an `euVehicleCategory`. Won't this condition try to update a tech-record's `euVehicleCategory` even if it already present?

Reply:
Ah, I see, you do this check in tech-records